oem/ibm/libpldmresponder/file_io_type_lid.hpp

```diff
@@ -169,12 +168,39 @@ class LidHandler : public FileHandler
                     oemIbmPlatformHandler = dynamic_cast<
                         pldm::responder::oem_ibm_platform::Handler*>(
                         oemPlatformHandler);
+                // rc = validate api;
+                auto& bus = pldm::utils::DBusHandler::getBus();
                 auto sensorId =
                     oemIbmPlatformHandler->codeUpdate->getMarkerLidSensor();
                 using namespace pldm::responder::oem_ibm_platform;
+                auto markerLidName = "80a00001.lid";
+                auto markerLidDirPath =
+                    fs::path(LID_STAGING_DIR) / "lid" / markerLidName;
+                try
```

Comment:
As Code changes are same in both the functions, can we avoid code duplication by using common function?

Reply:
done, good one.